File: OCEBook/app/src/main/java/com/devgrapher/ocebook/model/Page.java
package com.devgrapher.ocebook.model;

/**
 * Created by Brent on 2/16/17.
 */

public class Page {
    private final int mSpineItemPageIndex;
    private final int mSpineItemPageCount;
    private final String mIdref;
    private final int mSpineItemIndex;

    public Page(int spineItemPageIndex, int spineItemPageCount,
                String idref, int spineItemIndex) {
        mSpineItemPageIndex = spineItemPageIndex;
        mSpineItemPageCount = spineItemPageCount;
        mIdref = idref;
        mSpineItemIndex = spineItemIndex;
    }

    public int getSpineItemPageIndex() {
        return mSpineItemPageIndex;
    }

    public int getSpineItemPageCount() {
        return mSpineItemPageCount;
    }

    public String getIdref() {
        return mIdref;
    }

    public int getSpineItemIndex() {
        return mSpineItemIndex;
    }

    @Override
    public String toString() {
        return "Page [spineItemPageIndex=" + mSpineItemPageIndex
                + ", spineItemPageCount=" + mSpineItemPageCount + ", idref="
                + mIdref + ", spineItemIndex=" + mSpineItemIndex + "]";
    }
}

File: OCEBook/app/src/main/java/com/devgrapher/ocebook/model/Page.kt
package com.devgrapher.ocebook.model

/**
 * Created by Brent on 2/16/17.
 */

class Page(val spineItemPageIndex: Int, val spineItemPageCount: Int,
           val idref: String, val spineItemIndex: Int) {

    override fun toString(): String {
        return "Page [spineItemPageIndex=" + spineItemPageIndex +
                ", spineItemPageCount=" + spineItemPageCount + ", idref=" +
                idref + ", spineItemIndex=" + spineItemIndex + "]"
    }
}
